design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
+    name: String!
+    priceRange: ProductPriceRange!
+    url: ProductUrl
+    images: [ProductImage]
+    values: [ProductAttribute]
+}
+
+type ComparableIndicator {
+    code: String!
+    title: String!
+}
+
+type CompareList {
+    items: [ComparableItem]
+    indicators: [ComparableIndicator]
+}
+
+type Customer{
+    compareList: CompareList
```

Comment:
Looks like current schema does not support compare lists for guests. See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/product-compare.graphqls for possible solution.

Reply:
fixed